src/commcare_cloud/ansible/roles/couchdb2/defaults/main.yml

```diff
@@ -12,3 +12,4 @@ couchdb_backup_hours: 1
 couchdb_auth_cache_size: 200
 compact_couch_cron: False
 couch_backup_hourly: False
+couchdb_os_process_limit: 1000
```

Comment:
This value is based on the production value in _environments/production/public.yml_

Reply:
I think it probably doesn't hurt either way, but I agree there's no reason to change it in general. We're not even convinced it mattered in production, but it almost certainly doesn't matter on other environments, so I think we can continue to omit it.

@Charl1996 was the problem that something errored if this wasn't set? In that case I'd set it to the [couch default of 100](https://docs.couchdb.org/en/stable/config/query-servers.html#query_server_config/os_process_limit).

Reply:
@dannyroberts yes it complained that the default value was not found. I'll set it to 100 then :+1: